src/BackupChain.cs

```diff
@@ -4,13 +4,13 @@ namespace AgDatabaseMove
   using System.Collections.Generic;
   using System.IO;
   using System.Linq;
-  using Exceptions;
-  using SmoFacade;
+  using Microsoft.SqlServer.Management.Smo;

```

Comment:
smo should not be exposed from the Facade

Reply:
This isn't in the Facade and it's used for `DeviceType.Url` and `DeviceType.File`

Reply:
Exactly, outside the Facade we should not have references to SMO.

Reply:
We need to use this type somewhere... Do you have a suggestion on where to use these types inside of the Facade?  Another wrapper class for an Enum type?
Would fully qualifying the Enum instead of `using` satisfy these concerns?